ee/server/service/devices.go

```diff
@@ -208,7 +208,13 @@ func (svc *Service) validateReadyForLinuxEscrow(ctx context.Context, host *fleet
 		return &fleet.BadRequestError{Message: "Host's disk is not encrypted. Please enable disk encryption for this host."}
 	}
 
-	if host.OrbitVersion == nil || !fleet.IsAtLeastVersion(*host.OrbitVersion, fleet.MinOrbitLUKSVersion) {
+	// We have to pull Orbit info because the auth context doesn't fill in host.OrbitVersion
+	orbitInfo, err := svc.ds.GetHostOrbitInfo(ctx, host.ID)
```

Comment:
This means a user will have to wait for up to 1h after the fleetd upgrade to be able to perform the escrow (detail queries are updated every 1h)

Reply:
As mentioned on our call just now, `Create key` will show up as soon as they hit 4.60.0 if the following were true pre-upgrade:

1. Host is a supported Linux platform
2. Host is on a team with disk encryption as a policy
3. Host's disk is encrypted

Per the call, we'll keep the change but have a better error message when the version check fails.